### PR TITLE
[Feature #114296395] Change File storage to AWS for uploads

### PR DESCRIPTION
**Pull request by vickris**
Change file storage from Cloudinary to AWS.

### CODE REVIEW

File: tests/EpisodeSizeTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        $this->login();
+
+        $this->visit('dashboard/episode/create')
+             ->type('test', 'title')
+             ->type('Brief description', 'description')
+             ->attach('/audio/Haiti.mp3', 'podcast')
+             ->press('create')
+             ->assertEquals(0, count(Suyabay\Episode::where('episode_name', 'teststh')->get()));
+    }
+}
```
Comment (johnkariuki) [inline tests/EpisodeSizeTest.php:59]: Code Quality -  **Critical**

Write test to assert that a file between 1MB and 10MB is uploaded.

**Comment by unicodeveloper** [comment]
@andela-cvundi what's the status on this?

**Comment by vickris** [comment]
@unicodeveloper The upload works fine. Had one test misbehaving but will fix it. I have removed the other PR since we were still using Cloudinary when I made that PR. You can merge this PR. Currently working on tests together with John so I will revisit the misbehaving test.

**Comment by unicodeveloper** [comment]
Alright, I'll merge it for now

**Pull request merged**